README.md

```diff
@@ -55,6 +55,29 @@ test.all(common)
 
 A valid (read: that follows this interface) IPFS core implementation, must expose the following API.
 
+## Files
+
+### `Cat` 
+
+> Streams the data contained by an IPFS object(s) at the given IPFS multihash..
+
+##### `Go` **WIP**
+
+##### `JavaScript` - ipfs.cat(multihash, [callback])
+
+`multihash` is a [multihash][] which can be passed as
+
+- Buffer, the raw Buffer of the multihash (or of and encoded version)
+- String, the toString version of the multihash (or of an encoded version)
+
+`callback` must follow `function (err, stream) {}` signature, where `err` is an error if the operation was not successful and `stream` is a readable stream in object mode that contains objects representing a file in the form of { path: fileName, stream: fileData }.
```

Comment:
The core is returning a readable object stream from the exporter, which will send one object containing ```{ path: (the multihash provided to cat), stream: (a readable stream provided for that file) }```

Should I PR a change to the js-ipfs core here https://github.com/ipfs/js-ipfs/blob/master/src/core/ipfs/files.js#L48

to this 

```
const exportStream = Exporter(hash, self._dagS)
exportStream.on('data', (file) => {
      callback(null, file.stream)
})
```